docs/control-structures.rst

```diff
@@ -252,7 +252,7 @@ which only need to be created if there is a dispute.
             /// This complicated expression just tells you how the address
             /// can be pre-computed. It is just there for illustration.
             /// You actually only need ``new D{salt: salt}(arg)``.
-            address predictedAddress = address(bytes20(keccak256(abi.encodePacked(
+            address predictedAddress = address(uint(keccak256(abi.encodePacked(
```

Comment:
Don't you need to still truncate this to 160bits? `address(uint(bytes32(keccak256(..`

Reply:
keccak256 returns a bytes32 which can be converted to uint which can be trucated to address.

Reply:
I checked it on remix and it also compiles in the test.